src/main/scala/org/broadinstitute/dsde/workbench/sam/azure/AzureService.scala

```diff
@@ -0,0 +1,120 @@
+package org.broadinstitute.dsde.workbench.sam.azure
+
+import akka.http.scaladsl.model.StatusCodes
+import bio.terra.cloudres.azure.resourcemanager.common.Defaults
+import bio.terra.cloudres.azure.resourcemanager.msi.data.CreateUserAssignedManagedIdentityRequestData
+import cats.effect.IO
+import com.azure.core.management.Region
+import com.azure.core.util.Context
+import org.broadinstitute.dsde.workbench.model.{ErrorReport, WorkbenchExceptionWithErrorReport}
+import org.broadinstitute.dsde.workbench.sam._
+import org.broadinstitute.dsde.workbench.sam.dataAccess.DirectoryDAO
+import org.broadinstitute.dsde.workbench.sam.model._
+import org.broadinstitute.dsde.workbench.sam.util.SamRequestContext
+
+import scala.jdk.CollectionConverters._
+
+class AzureService(crlService: CrlService,
+                   directoryDAO: DirectoryDAO) {
+  // Static region in which to create all managed identities
```

Comment:
I can expand the comments. Basically the managed identity takes a region but it's not actually used -- the identity can still be assigned to resources across regions. So just using a static region instead of making the API take a region, etc. Will include links to Azure docs.